frontend/src/horses/AddHorseModal.tsx

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import React, {useState} from 'react';
+import Modal from 'react-modal';
+import {HorseModel} from "./HorseModel";
+import FieldLabelGroup from "../structuralComponents/FieldLabelGroup";
+import CloseIcon from "../icons/CloseIcon";
+import "../index/css/AddItemModal.css";
+import "../buttons/css/SubmitButton.css";
+
+type ModalProps = {
+    modalIsOpen: boolean,
+    closeModal: () => void,
+    reloadHorses: () => void,
+    setSuccessMessage: (input: string) => void,
+}
+
+function AddItemModal(props: ModalProps) {
+    const [newHorse, setNewHorse] = useState<HorseModel>({
+        id: "", name: "", owner: "", consumption: []
+    })
+
+
+    const saveNewHorse = () => {
+        axios.post("/horses/", newHorse)
+            .catch((e) => console.error("POST Error: " + e))
+            .then(props.reloadHorses)
+            .then(props.closeModal)
+            .then(() => setNewHorse({id: "", name: "", owner: "", consumption: []}))
+            .then(() => props.setSuccessMessage("Eintrag erfolgreich hinzugefügt"))
+
+    }
```

Comment:
you use same object  twice: {id: "", name: "", owner: "", consumption: []})
I prefer to create new variable and use it  as initial state or to reset .